### ISSUE TITLE
Feature Request: Alternate display methods for enums

### ISSUE THREAD

**Issue by tarngerine** [components-ai/css.gui#44]
Currently enums render as a `<select>` which is all fine and good, but I'd love an ability to render them as a radio group, too, which can be handy for a webflow style flex/grid layout widget

**Comment by johno** [comment]
Yeah I def think this is something that makes sense and is something we can implement. Perhaps the API could look something like:

```js
<Inputs.TextAlign inputType="radio" />
```
